### ISSUE TITLE
Can't start crawl on some URLs: "URL is not printable"

### ISSUE THREAD

**Issue by ivan** [ludios/grab-site#62]
```
~/.local/bin/grab-site http://www.eqmweekly.com.af/international/8288-سردرگمی-در-سیاست-خارجی-ترکیه --level=0 --no-sitemaps --concurrency=5 --1 --warc-max-size=524288000 --wpull-args="--no-check-certificate --timeout=300"
```

works but

```
~/.local/bin/grab-site http://www.eqmweekly.com.af/technology/8287-پشت-پرده‌‏ی-قتل-عام-در-نایجریا --level=0 --no-sitemaps --concurrency=5 --1 --warc-max-size=524288000 --wpull-args="--no-check-certificate --timeout=300"
```

outputs

```
Created lmdb db with map_size=1099511627776
Traceback (most recent call last):
  File "/home/grab/.local/bin/grab-site", line 4, in <module>
    main.main()
  File "/home/grab/.local/lib/python3.4/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/home/grab/.local/lib/python3.4/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/home/grab/.local/lib/python3.4/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/grab/.local/lib/python3.4/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/home/grab/.local/lib/python3.4/site-packages/libgrabsite/main.py", line 307, in main
    wpull.__main__.main()
  File "/home/grab/.local/lib/python3.4/site-packages/wpull/__main__.py", line 29, in main
    builder.build()
  File "/home/grab/.local/lib/python3.4/site-packages/wpull/builder.py", line 208, in build
    for url_info in self._build_input_urls():
  File "/home/grab/.local/lib/python3.4/site-packages/wpull/builder.py", line 524, in _build_input_urls
    url_string, default_scheme=default_scheme)
  File "/usr/lib/python3.4/functools.py", line 472, in wrapper
    result = user_function(*args, **kwds)
  File "/home/grab/.local/lib/python3.4/site-packages/wpull/url.py", line 128, in parse
    raise ValueError('URL is not printable: {}'.format(ascii(url)))
ValueError: URL is not printable: 'http://www.eqmweekly.com.af/technology/8287-\u067e\u0634\u062a-\u067e\u0631\u062f\u0647\u200c\u200f\u06cc-\u0642\u062a\u0644-\u0639\u0627\u0645-\u062f\u0631-\u0646\u0627\u06cc\u062c\u0631\u06cc\u0627'
```

**Comment by ivan** [comment]
Related to #11

**Comment by ivan** [comment]
This can be worked around by urlencoding the URLs first.  For example, by pasting them into a browser URL bar, hitting Enter, then copying the encoded URL out of the URL bar.

**Comment by ivan** [comment]
This is fixed as of 7c1afbefe0471e5a71059433f0d7b45c9f8d16ff / wpull 1.2.3
